packages/loaders/src/elements/Spinner.tsx

```diff
@@ -47,20 +47,19 @@ const computeFrames = (
 
 interface ISpinnerProps extends React.HTMLAttributes<SVGSVGElement> {
   /**
-   * Size of the loader. Can inherit from `font-size` styling.
+   * Sets the `height` and `width` in pixels. Inherits the parent `font-size` by default
```

Comment:
Should this be "Sets the _height_ and _width_ in pixels. Inherits the parent `font-size` by default"?

Reply:
...since we're not committing to using `height` and `width` CSS properties for sizing.

Reply:
Thanks for the fine tooth comb on this one. My eyes are going blind to back ticks.

Reply:
With fresh eyes, fixed up `Dots` to align with this as well.